src/python/pants/backend/go/goals/test_test.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+import pytest
+
+from pants.backend.go.goals.test import GoTestFieldSet
+from pants.backend.go.goals.test import rules as test_rules
+from pants.backend.go.target_types import GoModTarget, GoPackage
+from pants.build_graph.address import Address
+from pants.core.goals.test import TestResult
+from pants.engine.internals.scheduler import ExecutionError
+from pants.engine.rules import QueryRule
+from pants.testutil.rule_runner import RuleRunner
+
+
+@pytest.fixture
+def rule_runner() -> RuleRunner:
+    rule_runner = RuleRunner(
+        rules=[
+            *test_rules(),
+            QueryRule(TestResult, [GoTestFieldSet]),
+        ],
+        target_types=[GoPackage, GoModTarget],
+    )
+    rule_runner.set_options(["--backend-packages=pants.backend.experimental.go"])
```

Comment:
I realized last week that this isn't actually necessary. No need to fix in this PR though, you can save a tree